src/Powercord/plugins/pc-guilds/style.scss

```
@@ -73,6 +73,10 @@
   }
 }
 
+[data-react-beautiful-dnd-drag-handle="4"]{
```

This attribute may change at any time

---

Does the suggestions feature work?
I hope so
```suggestion
.pc-guild{
```

---

Because one space good
```suggestion
.pc-guild {
```